src/bin/miri.rs

```diff
@@ -6,14 +6,13 @@ extern crate rustc_driver;
 
 use miri::interpreter;
 use rustc::session::Session;
-use rustc_driver::{driver, CompilerCalls, Compilation};
+use rustc_driver::{driver, CompilerCalls};
 
 struct MiriCompilerCalls;
 
 impl<'a> CompilerCalls<'a> for MiriCompilerCalls {
     fn build_controller(&mut self, _: &Session) -> driver::CompileController<'a> {
         let mut control = driver::CompileController::basic();
-        control.after_analysis.stop = Compilation::Stop;
```

Comment:
Why did you remove this? Does this mean trans (and any other post-analysis steps) will run after Miri finishes?